front/app/containers/Admin/projects/project/analysis/Insights/Question.tsx

```diff
@@ -71,13 +71,18 @@ const Question = ({ insight }: Props) => {
     }
   };
 
+  const deleteTrailingIncompleteIDs = (str: string | null) => {
+    if (!str) return str;
+    return str.replace(/\[?[0-9a-f-]{0,35}$/, '');
+  };
+
   const replaceIdRefsWithLinks = (question) => {
     return reactStringReplace(
       question,
       /\[?([0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12})\]?/g,
       (match, i) => (
         <StyledButton onClick={() => setSelectedInputId(match)} key={i}>
-          <Icon name="search" />
+          <Icon name="idea" />
```

Comment:
This is great for ideation but for surveys might become a bit confusing.

Reply:
Totally agree, we need something better. First testing session said this was confusing, so just trying something else for now :)